### PR TITLE
[AL Journey] CSS Review

### PR DESCRIPTION
**Pull request by enzozccr**
https://app.clubhouse.io/enmarche/story/8559/parcours-al-ajuster-css-homepage

I need help on:
- fix CI
- add axis and modulos value in _nav.html.twig

### CODE REVIEW

File: templates/formation/_navbar.html.twig [hunk 1]
```diff
@@ -1,19 +1,33 @@
...
-            <a href="{{ path('app_formation_faq') }}" class="{{ is_active_route(app.request, 'app_formation_faq') ? 'active' }}">
-                Foire aux questions
-            </a>
-        </li>
-        <li>
-            <a href="#">
-                Faire le QCM
-            </a>
-        </li>
+        {% if app.request.get('_route') == 'app_formation_home' %}
```
Comment (enzozccr) [inline templates/formation/_navbar.html.twig]: Here we need something like `{% if not app.request.get('_route') == 'app_formation_article' %}`but i couldnt make it works

Comment (enzozccr) [inline templates/formation/_navbar.html.twig]:
```suggestion
        Here we need something like {% if not app.request.get('_route') == 'app_formation_article' %}but i couldnt make it works
```

File: templates/formation/_navbar.html.twig [hunk 2]
```diff
@@ -1,19 +1,33 @@
...
+            <li>
+                <a href="#">
+                    Faire le QCM
+                </a>
+            </li>
+        {% endif %}
+        {% if app.request.get('_route') == 'app_formation_article' %}
+            <li>
+                <a class="this-breadcrumb" href="{{ path('app_formation_home') }}">
+                    AXE [number]
```
Comment (enzozccr) [inline templates/formation/_navbar.html.twig]: Need to add axis nuber

File: templates/formation/_navbar.html.twig [hunk 3]
```diff
@@ -1,19 +1,33 @@
...
+                </a>
+            </li>
+            <li>
+                <span class="{{ is_active_route(app.request, 'app_formation_article') ? 'this-breadcrumb active' : 'this-breadcrumb'}}">
+                    [M.number] {{ article.title }}
```
Comment (enzozccr) [inline templates/formation/_navbar.html.twig]: Need to add module number

File: templates/formation/_navbar.html.twig (hunk 1 above)
Comment (nicoweb) [reply templates/formation/_navbar.html.twig]: you can use this: 
```
{% if is_active_route(app.request, 'app_formation_home') %}
```

File: src/DataFixtures/ORM/LoadFormationData.php [hunk 4]
```diff
@@ -30,13 +30,30 @@ public function load(ObjectManager $manager)
...
+        $manager->persist($axe1 = $this->createAxe('Mon comité et moi', $media));
+        $manager->persist($axe2 = $this->createAxe('Approfondissez vos connaissances', $media));
+        $manager->persist($axe3 = $this->createAxe('Faire un diagnostic territorial', $media));
+        $manager->persist($axe4 = $this->createAxe('Construisez votre projet territorial', $media));
 
-        $manager->persist($this->createArticle($axe1, 'Premier article du premier axe', $media));
-        $manager->persist($this->createArticle($axe1, 'Deuxième article du premier axe', $media));
-        $manager->persist($this->createArticle($axe2, 'Premier article du deuxième axe', $media));
-        $manager->persist($this->createArticle($axe2, 'Deuxième article du deuxième axe', $media));
+        $manager->persist($this->createArticle($axe1, 'Coordonner et animer une réunion efficace', $media));
```
Comment (nicoweb) [inline src/DataFixtures/ORM/LoadFormationData.php]: you've modified the fixtures, that's why the tests do not pass

**Pull request merged**